### PR TITLE
Fix search providers icons

### PR DESCRIPTION
**Pull request by Altahrim**
Related to advanced search

* Use white icon for user search provider
* Add missing icons for settings search providers

### CODE REVIEW

**Comment by Altahrim** [comment]
CI failure unrelated

File: lib/private/Search/SearchComposer.php [hunk 1]
```diff
@@ -221,6 +221,10 @@ private function fetchIcon(string $appId, string $providerId): string {
 			[$appId, 'app.svg'],
 			['core', 'places/default-app-icon.svg'],
 		];
+		if ($appId === 'settings' && $providerId === 'users') {
+			// The file /apps/settings/users.svg if already used in black version by user menu
```
Comment (come-nc) [inline lib/private/Search/SearchComposer.php:225]:
```suggestion
			// The file /apps/settings/users.svg is already used in black version by user menu
```
Typo maybe? Or I do not understand the comment.
But still I do not understand the comment I think.

Comment (Altahrim) [reply lib/private/Search/SearchComposer.php:225]: Yes, typo…
We are using `/apps/settings/users.svg` as an icon in the user menu on top right. In the menu, we need a black icon.
But for search, we need a white version.

Comment (Fenn-CS) [reply lib/private/Search/SearchComposer.php:225]: Must these icons, have the same name? Can't both colors be present? to avoid this?

Comment (Altahrim) [reply lib/private/Search/SearchComposer.php:225]: > Must these icons have the same name?

My hack avoid this. The new icon is white and we keep the old one

**Review by come-nc — APPROVED** [review]

**Pull request merged**